lib/puppet/http/service/puppetserver.rb

```diff
@@ -26,13 +26,24 @@ def initialize(client, session, server, port)
   # @api private
   #
   def get_simple_status(ssl_context: nil)
-    response = @client.get(
-      with_base_url("/status/v1/simple/master"),
-      headers: add_puppet_headers({}),
-      options: {ssl_context: ssl_context}
-    )
+    request_path = "/status/v1/simple/server"
```

Comment:
Making the first request to `/status/v1/simple/server` will double the response time for old puppetservers that do not have this endpoint.